en/faq/dokku-deployment.md

```diff
@@ -20,7 +18,7 @@ dokku config:set my-nuxt-app NPM_CONFIG_PRODUCTION=false
 Also, we want our application to listen on the port `0.0.0.0` and run in production mode:
 ```bash
 // on Dokku Server
-dokku config:set my-nuxt-app HOST=0.0.0.0 NODE_ENV=production
+dokku config:set nuxt-app HOST=0.0.0.0 NODE_ENV=production
```

Comment:
@Atinux 

Here should be my-nuxt-app .

because that is app created name on Dokku
`dokku config:set my-nuxt-app NPM_CONFIG_PRODUCTION=false`
it's a same app name 
```
dokku config:set my-nuxt-app NPM_CONFIG_PRODUCTION=false
dokku config:set my-nuxt-app HOST=0.0.0.0 NODE_ENV=production
```
also same this
```
dokku config:set my-nuxt-app NPM_CONFIG_PRODUCTION=false HOST=0.0.0.0 NODE_ENV=production
```

thanks.